third_party/nvfuser/runtime/complex_number.cu

```diff
@@ -0,0 +1,1313 @@
+#ifndef __NVCC__
+#define POS_INFINITY __int_as_float(0x7f800000)
+#define INFINITY POS_INFINITY
+#define NEG_INFINITY __int_as_float(0xff800000)
+#define NAN __int_as_float(0x7fffffff)
+
+namespace std {
```

Comment:
Same here, should we put a copyright section and a pointer to the original code here for LLVM copied code?

Reply:
added